storage/pruning/fullHistoryPruningStorer.go

```diff
@@ -96,11 +96,46 @@ func (fhps *FullHistoryPruningStorer) searchInEpoch(key []byte, epoch uint32) ([
 }
 
 func (fhps *FullHistoryPruningStorer) getFromOldEpoch(key []byte, epoch uint32) ([]byte, error) {
+	persister, err := fhps.getPersister(epoch)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := persister.Get(key)
+	if err == nil {
+		return res, nil
+	}
+
+	log.Warn("GetFromEpoch persister",
```

Comment:
I would keep this as maximum to log.Debug (maybe log.Trace?), otherwise there is an attack vector: I connect to a history node and request data from a very-large epoch